code/src/index.css

```diff
@@ -1,13 +1,89 @@
 body {
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
-    sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  font-family: 'Roboto', sans-serif;
 }
 
-code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
-    monospace;
+/* Submit */
+
+.submit-container {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 300px;
+
+}
+
+.submit-form {
+  display: flex;
+  flex-direction: column;
+  width: 80%;
+  height: 200px;
+  border:solid 2px #000;
+  background-color: #c9d5e1c1;
+  box-shadow: 5px 5px;
+  padding: 15px;
+}
+
+input[type=text] {
+  width: 100%;
+  height: 80px;
+}
+
+input[type=submit] {
+  border-radius: 25px;
+  width: 50%;
+  height: 40px;
+  margin: 5px;
+  background-color:#ffadad;
+}
+
+/* Messages */
+
+.message-container {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-direction: column;
 }
+
+.message {
+  width: 90%;
+  height: 150px;
+  border:solid 2px #000;
+  background-color: #c9d5e1c1;
+  box-shadow: 5px 5px;
+  margin-top: 50px;
+  font-size: 12px;
+  padding: 5px;
+}
```

Comment:
Here you could add a 'max-width' of maybe 400px so the boxes don't get so stretched out on a big screen. Same with the form.